easybuild/easyconfigs/c/CrayPGI/CrayPGI-2016.04.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayPGI'
+version = '2016.04'
+
+homepage = 'http://www.pgroup.com/'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-pgi module."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+dependencies = [
+     # PrgEnv version is not pinned, as Cray recommends to use the latest (default) version
+     # ('PrgEnv-pgi/' + version, EXTERNAL_MODULE),
+     ('PrgEnv-pgi/5.2.82', EXTERNAL_MODULE),
+     ('pgi/16.3.0' , EXTERNAL_MODULE),
+     ('cray-mpich/7.3.2', EXTERNAL_MODULE),
+     # ('cray-libsci/16.03.1', EXTERNAL_MODULE), # Cray does not provide cray-libsci for PrgEnv-pgi
```

Comment:
hmm, this is going to yield problems whenever BLAS/LAPACK is required?

@gppezzi